mautic/mautic
# Tooltip help for searching segments is inaccurate

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | N
| Automated tests included? | N
| Related user documentation PR URL |  N
| Related developer documentation PR URL |  N
| Issues addressed (#s or URLs) |
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: 
1. We've removed commands 
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Go to Segments page.
2. Try to type "is:" in the search field. You will see some commands we don't want to use (is:mine, is:inactive).

#### Steps to test this PR:
1. Go to Segments page.
2. Open Developer menu (or developer panel) in your browser and clear the following local storage entry: __/index.php/s/ajax?action=commandList&model=lead.list__data
3. Reload the page
4. You should see that those unwanted search commands are gone.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
